src/parser/jobs/rdm/modules/ManaStackGauge.tsx

```diff
@@ -0,0 +1,214 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import {JOBS} from 'data/JOBS'
+import {Event, Events} from 'event'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {CounterGauge, Gauge as CoreGauge} from 'parser/core/modules/Gauge'
+import {Statistics} from 'parser/core/modules/Statistics'
+import Suggestions, {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {Gauge} from 'parser/jobs/rdm/modules/Gauge'
+import {DualStatistic} from 'parser/jobs/rdm/statistics/DualStatistic'
+import React, {Fragment} from 'react'
+
+interface ManaStackGaugeModifier {
+	manaStack: number
+}
+interface GaugeBreaker {
+	action: number,
+	timestamp: number,
+	manaStacksLost: number,
+}
+export const MINIMUM = 0
+export const MAXIMUM = 3
+
+export class ManaStackGauge extends CoreGauge {
+	static override handle = 'manaStackGauge'
+	static override title = t('rdm.manaStackGauge.title')`Mana Stack Gauge Usage`
+	static override debug = false
+
+	@dependency private suggestions!: Suggestions
+	@dependency private statistics!: Statistics
+	@dependency private gauge!: Gauge
+
+	private manaStackGauge = this.add(new CounterGauge({
+		graph: {
+			label: <Trans id="rdm.manaStackGauge.resource.manaStacks">Mana Stacks</Trans>,
+			color: JOBS.RED_MAGE.colour,
+		},
+		maximum: MAXIMUM,
+		minimum: MINIMUM,
+	}))
+	public gaugeModifiers = new Map<number, ManaStackGaugeModifier>([
+		[this.data.actions.ENCHANTED_REPRISE.id, {manaStack: 1}],
+		[this.data.actions.ENCHANTED_MOULINET.id, {manaStack: 1}],
+		[this.data.actions.ENCHANTED_RIPOSTE.id, {manaStack: 1}],
+		[this.data.actions.ENCHANTED_ZWERCHHAU.id, {manaStack: 1}],
+		[this.data.actions.ENCHANTED_REDOUBLEMENT.id, {manaStack: 1}],
+	])
+	public spenderModifiers = new Map<number, ManaStackGaugeModifier>([
+		[this.data.actions.VERHOLY.id, {manaStack: -1}],
+		[this.data.actions.VERFLARE.id, {manaStack: -1}],
+		[this.data.actions.SCORCH.id, {manaStack: -1}],
+		[this.data.actions.RESOLUTION.id, {manaStack: -1}],
+	])
+	public gaugeBreakers: number[] = [
+		this.data.actions.VERAERO.id,
+		this.data.actions.VERAERO_II.id,
+		this.data.actions.VERAERO_III.id,
+		this.data.actions.VERSTONE.id,
+		this.data.actions.VERTHUNDER.id,
+		this.data.actions.VERTHUNDER_II.id,
+		this.data.actions.VERTHUNDER_III.id,
+		this.data.actions.VERFIRE.id,
+		this.data.actions.JOLT.id,
+		this.data.actions.JOLT_II.id,
+		this.data.actions.IMPACT.id,
+	]
+	gaugeBreak: GaugeBreaker[] = []
+	//Any loss is a major severity
+	severity = {
+		1: SEVERITY.MAJOR,
+	}
+	manaStatistics = {
+		white: {
+			overcapLoss: 0,
+		},
+		black: {
+			overcapLoss: 0,
+		},
+	}
```

Comment:
I assume this is a carry-over from the other gauge file?

Reply:
No this is meant to track when you overcap your mana stack that mana you wasted in overcapping as that mana is lost, you gained nothing for it beyond not losing the chance to do the finisher combo.

Reply:
Got it. Thanks.